test/test.js

```diff
@@ -83,6 +85,24 @@ plantuml::test/fixtures/alice.puml[svg,role=sequence]
       const hash = rusha.createHash().update(`https://kroki.io/plantuml/svg/${encode(file)}`).digest('hex')
       expect(html).to.contain(`<img src=".asciidoctor/kroki/diag-${hash}.svg" alt="diagram">`)
     })
+    it('should create diagrams in imagesdir if kroki-fetch-diagram is set', function (done) {
+      const registry = asciidoctor.Extensions.create()
+      asciidoctorKroki.register(registry)
+      const file = `${__dirname}/fixtures/fetch/doc.adoc`
+      const doc = asciidoctor.convertFile(file, { extension_registry: registry, safe: 'unsafe' })
+      fs.unlinkSync(doc.getAttributes().outfile)
+      const imageLocation = path.join(doc.base_dir, doc.getAttributes().imagesdir)
+      fs.readdir(imageLocation, (err, files) => {
```

Comment:
I think we should use `fsPromises.readdir` https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fspromises_readdir_path_options

```suggestion
      const files = await fsPromises.readdir(imageLocation)
```